source/MapDetailPanel.cpp

```diff
@@ -573,13 +573,29 @@ void MapDetailPanel::DrawKey()
 		// four largest visible governments are labeled in the legend.
 		vector<pair<double, const Government *>> distances;
 		for(const auto &it : closeGovernments)
+		{
+			if(!it.first)
+				continue;
```

Comment:
Why this?

Reply:
Peter suggested it and it seems like a reasonable enough thing to do.

Reply:
oh nvm I thought that was the distance, carry on ~~I need to sleep~~